plugin/trino-elasticsearch/src/main/java/io/trino/plugin/elasticsearch/decoders/TimestampDecoder.java

```diff
@@ -68,16 +78,18 @@ public void decode(SearchHit hit, Supplier<Object> getter, BlockBuilder output)
             LocalDateTime timestamp;
             if (value instanceof String) {
                 String valueString = (String) value;
-                Long epochMillis = Longs.tryParse(valueString);
-                if (epochMillis != null) {
-                    timestamp = LocalDateTime.ofInstant(Instant.ofEpochMilli(epochMillis), UTC);
+                try {
+                    timestamp = LocalDateTime.from(formatter.parse(valueString));
                 }
-                else {
+                catch (IllegalArgumentException e) {
```

Comment:
`org.elasticsearch.common.time.DateFormatter#parse` may throw `DateTimeException`. Also, can you add tests to cover the catch block?

Reply:
I found the version of elasticsearch dependency is 6.8.23. Default, it use JodaDateFormatter which throw IllegalArgumentException and JavaDateFormtatter will throw DateTimeException. I will catch both of them. 

BTW do we consider to upgrade the depedency version?